tests/x25519.cc

```diff
@@ -10,11 +10,11 @@
 #include <gtest/gtest.h>
 #include <nlohmann/json.hpp>
 
+#include "../assets/test_vectors/x25519/curve25519_vectors.h"
```

Comment:
Don't use relative include paths like this. Its not portable.

Reply:
I'll convert that to JSON to get rid of the header.